src/views/Icons/Icons.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+import './Icons.css';
+import {SET_COMPONENT_MARKUP} from '../../actions/atomic';
+
+class Icons extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      icons: window.icons,
```

Comment:
Maybe we can add this logic in a proper Redux collection if it usefull somewhere else.